.prettierignore

```diff
@@ -0,0 +1,16 @@
+/packages/**/node_modules
+/packages/**/dist
+/packages/**/es2015
+/packages/**/coverage
+/packages/marshal-formularious
```

Comment:
👀 I guess that all private projects references should be removed

Reply:
removed!